### PR TITLE
Sequence classification datasets and experiments

### PR DESCRIPTION
**Pull request by marionleborgne**

### CODE REVIEW

**Comment by marionleborgne** [comment]
@subutai this is the sequence classification work described in my email

File: classification/generate_model_params.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+from model_params.template_model_params import MODEL_PARAMS as TEMPLATE_MODEL_PARAMS
```
Comment (subutai) [inline classification/generate_model_params.py]: Need copyright statement at top.

File: classification/plot.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import csv
```
Comment (subutai) [inline classification/plot.py]: Copyright statement

File: classification/sensortag.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
+var util = require('util');
```
Comment (subutai) [inline classification/sensortag.js]: Copyright statement

File: classification/settings.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# start and end indices of each class (inclusive)
```
Comment (subutai) [inline classification/settings.py]: Copyright statement

Comment (marionleborgne) [inline classification/settings.py]: added

File: classification/sensortag.js (hunk 3 above)
Comment (marionleborgne) [inline classification/sensortag.js]: done

File: classification/generate_model_params.py (hunk 1 above)
Comment (marionleborgne) [inline classification/generate_model_params.py]: done

**Comment by marionleborgne** [comment]
@subutai I added the missing copyright statements. Thanks.

**Pull request merged**